File: html/getSchedule.php
<?php
$i = intval($_GET('i'));
$s = inval($_GET('s'));

$con = mysqli_connect('localhost','root','password','Schedule');
if (!$con) (
    die('Could not connect: ' .mysqli_error($con));
)

mysqli_select_db($con,"ajax_demo");
$sql="SELECT schedules FROM dotslash WHERE id = '".$i."'";
$json = mysqli_query($con,$sql);

$data = json_decode ($json);
echo $data[strval($s)];
mysqli_close($con);
?>

File: html/getSchedule.py
import json
from flaskext.mysql import MySQL

def accessDatabase(db_id,mysql):
    conn = mysql.connect()
    cur = conn.cursor()
    cur.execute("SELECT schedules FROM dotslash WHERE id=\'"+str(db_id)+"\'")
    schedules = cur.fetchall()[0][0]
    cur.close()
    conn.close()
    return schedules

def convertToJSON(s_index, db_id, mysql):
    schedules = accessDatabase(db_id,mysql)
    choice = json.loads(schedules)['schedules']['S'+str(s_index)]
    return json.dumps(choice)

"""
db = MySQLdb.connect(host="localhost",    # your host, usually localhost
                     user="root",         # your username
                     passwd="password",  # your password
                     db="Schedule")        # name of the data base

cursor = db.cursor()
cursor.execute("SELECT schedules FROM dotslash WHERE id=\'"+str(26)+"\'")
"""